src/media/js/site/components/conditionalReverseLink.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {ReverseLink} from 'react-router-reverse';
+import urlJoin from 'url-join';
+
+import {checkPermissions} from '../login';
+
+
+export class ConditionalReverseLink extends React.Component {
+  static propTypes = {
+    children: React.PropTypes.node.isRequired,
+    permissions: React.PropTypes.array.isRequired,
+    user: React.PropTypes.object.isRequired,
+    to: React.PropTypes.string.isRequired,
+  };
+
+  _checkPermissions() {
```

Comment:
I did that because https://github.com/mozilla/marketplace-content-tools/pull/56#discussion-diff-41573283R6